packages/utils/src/CypressUtils/TableUtils.ts

```diff
@@ -0,0 +1,207 @@
+  columnName: string, // - identifier of the column to be used to validate data
+  tableHeaders: string[], // - - header of the table
+  emptyStateTitle: string, // - title to check on empty state
+  validateURL: boolean, //- whether to validate URL parameters
+  hasDefaultFilters: boolean //- whether default filters exist or not
+) {
+  if (hasDefaultFilters) {
+    removeAllChips();
+  }
+  applyFilters(filters, filtersConf);
+
+  if (values.length === 0) {
+    checkEmptyState(emptyStateTitle);
+    checkTableHeaders(tableHeaders);
+  } else {
+    cy.get(`td[data-label="${columnName}"]`)
+      .then(($els) => {
+        return _.map(Cypress.$.makeArray($els), 'innerText');
+      })
+      .should('deep.equal', values);
+  }
+
+  // validate chips and url params
+  cy.get(CHIP_GROUP)
+    .should('have.length', Object.keys(filters).length)
+    .then(() => {
+      if (validateURL) {
+        for (const [k, v] of Object.entries(filtersConf)) {
+          if (k in filters) {
+            // @ts-ignore
+            // NEED TO FIX type error here
+            const urlValue = v.urlValue(filters[k]);
+            expect(window.location.search).to.contain(`${v.urlParam}=${urlValue}`);
+          } else {
+            expect(window.location.search).to.not.contain(`${v.urlParam}=`);
+          }
+        }
+      }
+    });
+
+  // check chips
+  for (const [k, v] of Object.entries(filters)) {
+    // NEED TO FIX type error here
+    // @ts-ignore
+    const groupName = filtersConf[k].selectorText;
+    // @ts-ignore
+    const nExpectedItems = filtersConf[k].type === 'checkbox' ? v.length : 1;
+    cy.get(CHIP_GROUP)
+      .contains(groupName)
+      .parents(CHIP_GROUP)
+      .then((chipGroup) => {
+        cy.wrap(chipGroup).find(CHIP).its('length').should('be.eq', Math.min(3, nExpectedItems)); // limited to show 3
+      });
+  }
+  cy.get('button').contains('Reset filters').should('exist');
+}
+
+//- following order from API response and complemented with any
+//modification needed
+interface ClusterData {
+  cluster: string;
+  cluster_name: string;
+  last_checked: string;
+  meta: {
+    cluster_version: string;
+  };
+  impacted: string;
+  name: string;
+}
+//A column is sorted and then data and a reference column (which can be different than the
+//sorting one) are compared to see if order matches
+/**
+ * @param {*} sortingField - key in the data to sort by, or function
+ * @param {string} label - identifier of the column
+ * @param {string} order - ascending or descending
+ * @param {string} columnField - identifier of the column to be used in the comparison with the data
+ * @param {string} dataField - key in the data to be used in the comparison with the data
+ * @param {integer} nExpectedRows - number of expected rows to be displayed in the table
+ * @param {string} validateURL - string expected as sort value in URL. If not provided, it is not checked
+ */
+export function checkSorting(
```

Comment:
I tried to make it work for RHEL Advisor, but they utilize the sorting and filtering on the backend. I would remove CheckSorting and CheckFiltering from this PR because they won't be utilized

Reply:
yes please